### PR TITLE
zuul integration with the repository

### LINKED ISSUE #5: zuul integration
please integrate with zuul and run at least the yaml linter in check and gate pipelines.

### PR DESCRIPTION
**Pull request by harshad16**
zuul integration with the repository
Fixes: #5 
Related-To: https://github.com/thoth-station/zuul-config/pull/85
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

**Review by goern — APPROVED** [review]

File: .coafile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+[all]
+bears = LineCountBear, FilenameBear
+files = **.py, **.yaml, **.toml, **.rst, **.md
+file_naming_convention = snake
+ignore = **/__pycache__/**, **/__pycache__, __pycache__, __pycache__/**, **/*.pyc, *.pyc
+max_line_length = 120
+max_lines_per_file = 2000
+
+[all.python]
```
Comment (CermakM) [inline .coafile]: Thanks, @harshad16 ! Just a note to the python linters, I don't think that they are necessary, I don't see this being a python package. WDYT @fridex? (Anyway, it doesn't make much difference to have them there or not to)

Comment (fridex) [reply .coafile]: +1, this is probably a copy-paste configuration.

BTW we could think about organization-wide style and configure linters accordingly (so we don't need to place config files in each repo).

Comment (CermakM) [reply .coafile]: Yep, I think that the template repo would be a nice thing to have.

Comment (CermakM) [reply .coafile]: Or we could be more adventurous and let Kebechet generate the configuration according to the language of the repo? @fridex

Comment (fridex) [reply .coafile]: I would probably move this to zuul CI so we do not maintain these files all over the place but rather have one configuration which is automatically used in CI on a repo.

Comment (CermakM) [reply .coafile]: Ack.

Comment (harshad16) [reply .coafile]: @CermakM I had just copy-pasted with respect to other repos, I will remove it if python is not being used here. :)

**Pull request merged**